wimm.Guardian/wimm.Guardian.csproj

```diff
@@ -1,28 +1,28 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;net45</TargetFrameworks>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
```

Comment:
I noticed these versions aren't in sync anymore but it still builds both. Not going to block on this since we're still in 0.x, maybe we'll just remove the .NET framework target altogether later.